librz/analysis/meta.c

```diff
@@ -348,7 +348,7 @@ RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64
 				if (!s) {
 					s = strdup(pstr);
 				}
-				if (rad) {
+				if (mode) {
```

Comment:
I don't think this should be `if (mode)`, code doesn't make sense in JSON mode, for example. Lets just use some specific mode here.